torch/nn/_intrinsic/fake_quantize.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from ..modules.module import Module
+from ..._jit_internal import weak_module
+from ...autograd.function import Function
+
+@weak_module
+class DefaultObserverModule(Module):
```

Comment:
Let us have all observer modules in observer.py and use the same defaults for fake quant and for post training quant.